matter/si91x/siwx917/BRD4338A/autogen/sl_component_catalog.h

```diff
@@ -24,10 +24,8 @@
 #define SL_CATALOG_MEMORY_MANAGER_PRESENT
 #define SL_CATALOG_NVM3_PRESENT
 #define SL_CATALOG_SEGGER_RTT_PRESENT
-#ifdef DISPLAY_ENABLED
+#ifdef DISPLAY_ENABLED || SL_ICD_ENABLED
 #define SL_CATALOG_SLEEPTIMER_PRESENT
-#define SL_CATALOG_DMD_MEMLCD_PRESENT

```

Comment:
is LCD support removed or is this not needed for 917 anymore

Reply:
It was needed, added back.